egs/wham/DynamicMixing/model.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+import torch
+from torch import nn
+
+import asteroid.filterbanks as fb
+from asteroid import torch_utils
+from asteroid.masknn import DPRNN
+from asteroid.engine.optimizers import make_optimizer
+
+
+class Model(nn.Module):
+    def __init__(self, encoder, masker, decoder):
+        super().__init__()
+        self.encoder = encoder
+        self.masker = masker
+        self.decoder = decoder
+
+    def forward(self, x):
+        if len(x.shape) == 2:
+            x = x.unsqueeze(1)
+        tf_rep = self.encoder(x)
+        est_masks = self.masker(tf_rep)
+        masked_tf_rep = est_masks * tf_rep.unsqueeze(1)
+        return self.pad_output_to_inp(self.decoder(masked_tf_rep), x)
+
+    @staticmethod
+    def pad_output_to_inp(output, inp):
+        """ Pad first argument to have same size as second argument"""
+        inp_len = inp.size(-1)
+        output_len = output.size(-1)
+        return nn.functional.pad(output, [0, inp_len - output_len])
```

Comment:
Can you use the padding function that's used in the DPRNN recipe, it has changed